analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotShiftByZeroOrIntSize.cs

```diff
@@ -197,28 +194,28 @@ private static string FindProblemDescription(int typeSizeInBits, int shiftBy, Sh
 
             if (shiftDirection == Shift.Right)
             {
-                return string.Format(MessageFormat_RightShiftTooLarge, shiftBy);
+                return string.Format(MessageFormatRightShiftTooLarge, shiftBy);
             }
 
             var shiftSuggestion = shiftBy % typeSizeInBits;
 
             if (typeSizeInBits == 64)
             {
                 return shiftSuggestion == 0
-                    ? string.Format(MessageFormat_UselessShift, shiftBy)
-                    : string.Format(MessageFormat_ShiftTooLarge, shiftSuggestion);
+                    ? string.Format(MessageFormatUselessShift, shiftBy)
+                    : string.Format(MessageFormatShiftTooLarge, shiftSuggestion);
             }
 
             if (shiftSuggestion == 0)
             {
-                return string.Format(MessageFormat_UseLargerTypeOrPromote,
+                return string.Format(MessageFormatUseLargerTypeOrPromote,
                     "less than " + typeSizeInBits);
             }
 
-            return string.Format(MessageFormat_UseLargerTypeOrPromote, shiftSuggestion);
+            return string.Format(MessageFormatUseLargerTypeOrPromote, shiftSuggestion);
         }
 
-        private class ShiftInstance
+        private sealed class ShiftInstance
         {
```

Comment:
The `public ShiftInstance(SyntaxNode node)` constructor can be turned into an expression-bodied constructor.